### ISSUE TITLE
Malformed lines in translation files

### ISSUE THREAD

**Issue by Robyt3** [ddnet/ddnet#6946]
The following log messages are printed when selecting all languages with `loglevel 3` or higher:

```
localization: loaded 'languages/bosnian.txt'
localization: loaded 'languages/catalan.txt'
localization: malform replacement line (1257) for ''
localization: loaded 'languages/danish.txt'
localization: malform replacement line (1151) for ''
localization: loaded 'languages/german.txt'
localization: loaded 'languages/spanish.txt'
localization: malform replacement line (1598) for ''
localization: loaded 'languages/esperanto.txt'
localization: malform replacement line (571) for ''
localization: loaded 'languages/french.txt'
localization: malform replacement line (1540) for ''
localization: loaded 'languages/galician.txt'
localization: malform replacement line (1619) for ''
localization: loaded 'languages/italian.txt'
localization: malform replacement line (722) for 'transmits your player name to info.ddnet.org'
localization: malform replacement line (724) for '== Trasmette il tuo nome da giocatore a info.ddnet.org'
localization: malform replacement line (1169) for ''
localization: loaded 'languages/hungarian.txt'
localization: malform replacement line (1486) for ''
localization: loaded 'languages/dutch.txt'
localization: malform replacement line (1306) for ''
localization: loaded 'languages/norwegian.txt'
localization: malform replacement line (1152) for ''
localization: loaded 'languages/persian.txt'
localization: malform replacement line (868) for ''
localization: loaded 'languages/polish.txt'
localization: malform replacement line (1298) for ''
localization: loaded 'languages/portuguese.txt'
localization: malform replacement line (936) for ''
localization: loaded 'languages/brazilian_portuguese.txt'
localization: malform replacement line (1601) for ''
localization: loaded 'languages/romanian.txt'
localization: malform replacement line (595) for ''
localization: loaded 'languages/slovak.txt'
localization: malform replacement line (583) for ''
localization: loaded 'languages/serbian.txt'
localization: malform replacement line (1487) for ''
localization: loaded 'languages/finnish.txt'
localization: malform replacement line (1138) for ''
localization: loaded 'languages/swedish.txt'
localization: malform replacement line (1239) for ''
localization: loaded 'languages/turkish.txt'
localization: malform replacement line (1047) for ''
localization: loaded 'languages/czech.txt'
localization: malform replacement line (1153) for ''
localization: loaded 'languages/greek.txt'
localization: malform replacement line (589) for ''
localization: loaded 'languages/belarusian.txt'
localization: malform replacement line (583) for ''
localization: malform replacement line (1409) for ''
localization: malform replacement line (1411) for '== Уключыць запісы'
localization: loaded 'languages/bulgarian.txt'
localization: malform replacement line (580) for ''
localization: loaded 'languages/kyrgyz.txt'
localization: malform replacement line (580) for ''
localization: loaded 'languages/russian.txt'
localization: loaded 'languages/serbian_cyrillic.txt'
localization: malform replacement line (1486) for ''
localization: loaded 'languages/ukrainian.txt'
localization: malform replacement line (1222) for ''
localization: loaded 'languages/chuvash.txt'
localization: malform replacement line (583) for ''
localization: loaded 'languages/japanese.txt'
localization: malform replacement line (1192) for ''
localization: loaded 'languages/simplified_chinese.txt'
localization: malform replacement line (1614) for ''
localization: loaded 'languages/traditional_chinese.txt'
localization: malform replacement line (1603) for ''
localization: loaded 'languages/korean.txt'
localization: malform replacement line (1555) for ''
localization: loaded 'languages/arabic.txt'
localization: malform replacement line (1143) for ''
```

(error messages are printed after the filename of the respective language file)

(also, it should say `malformed` not `malform` in the error message)

**Comment by Vy0x2** [comment]
I can clean up a bit, but cant assign myself

**Comment by Vy0x2** [comment]
Fixed by #6948
